contracts/bridge_test.go

```diff
@@ -371,3 +380,96 @@ func TestBridgePublicVariables(t *testing.T) {
 	rnonce, err := b.RequestNonce(nil)
 	assert.Equal(t, uint64(0), rnonce)
 }
+
+// TestExtendedBridgeAndCallback checks the following:
+// - the extBridge can make contract call to callback contract.
+func TestExtendedBridgeAndCallback(t *testing.T) {
+	bridgeAccountKey, _ := crypto.GenerateKey()
+	bridgeAccount := bind.NewKeyedTransactor(bridgeAccountKey)
+
+	//aliceKey, _ := crypto.GenerateKey()
+	//aliceAcc := bind.NewKeyedTransactor(aliceKey)
+
+	bobKey, _ := crypto.GenerateKey()
+	bobAcc := bind.NewKeyedTransactor(bobKey)
+
+	alloc := blockchain.GenesisAlloc{bridgeAccount.From: {Balance: big.NewInt(params.KLAY)}}
+	backend := backends.NewSimulatedBackend(alloc)
+
+	// Deploy extBridge
+	bridgeAddr, tx, eb, err := extbridge.DeployExtBridge(bridgeAccount, backend, true)
+	assert.NoError(t, err)
+	backend.Commit()
+	assert.Nil(t, WaitMined(tx, backend, t))
+
+	// Deploy token
+	erc20Addr, tx, erc20, err := sctoken.DeployServiceChainToken(bridgeAccount, backend, bridgeAddr)
+	assert.NoError(t, err)
+	backend.Commit()
+	assert.Nil(t, WaitMined(tx, backend, t))
+
+	// Give minter role to bridge contract
+	tx, err = erc20.AddMinter(bridgeAccount, bridgeAddr)
```

Comment:
Can owner be minter automatically while construction?

Reply:
Yes we can do that if we modifiy sctoken code. I will try it. :)